ui/pages/confirmation/confirmation.js

```diff
@@ -185,23 +184,20 @@ export default function ConfirmationPage() {
         </div>
       )}
       <div className="confirmation-page__content">
-        <Box justifyContent="center">
-          <NetworkDisplay
-            colored={false}
-            indicatorSize={SIZES.XS}
-            labelProps={{ color: COLORS.BLACK }}
+        <Box
+          alignItems="center"
+          marginTop={1}
+          padding={[1, 4, 4]}
+          flexDirection={FLEX_DIRECTION.COLUMN}
+        >
+          <SiteIcon
+            icon={originMetadata.iconUrl}
+            name={originMetadata.hostname}
+            size={36}
           />
-        </Box>
-        <Box justifyContent="center" padding={[1, 4, 4]}>
           <Chip
             label={stripHttpsScheme(originMetadata.origin)}
-            leftIcon={
-              <SiteIcon
-                icon={originMetadata.iconUrl}
-                name={originMetadata.hostname}
-                size={32}
-              />
-            }
+            borderColor={COLORS.WHITE}
```

Comment:
do not change these styles just yet, we may move to a place where no confirmations need these but until then if you rebase on develop you should see the chip has been removed and a template prop added to control the display of the network display component.

Reply:
I resolve this conflicts as agreed in this slack conversation. 
https://consensys.slack.com/archives/C02GH833686/p1643894366164449